tests/components/online_image/common.yaml

```
@@ -26,6 +26,10 @@ online_image:
     format: PNG
     type: RGB
     transparency: chroma_key
+  - id: online_binary_bmp
+    url: http://www.libpng.org/pub/png/img_png/pnglogo-blk-tiny.png
```

It's not really executed, so it's not that important, but it would feel more consistent, if the URL would point to a bmp image ;-)

---

Haha yes. I just didnt know any and was being lazy at the end.

---

```suggestion
    url: https://samples-files.com/samples/images/bmp/480-360-sample.bmp
```
As mentioned, feel free to ignore (nobody knows, how long the link will be valid); but it feels better to have an actual BMP link ;-)